llvm/lib/DWARFLinkerParallel/DWARFEmitterImpl.cpp

```diff
@@ -230,9 +230,17 @@ void DwarfEmitterImpl::emitDebugNames(DWARF5AccelTable &Table,
     return;
 
   Asm->OutStreamer->switchSection(MOFI->getDwarfDebugNamesSection());
+  dwarf::Form Form =
+      DIEInteger::BestForm(/*IsSigned*/ false, (uint64_t)CUidToIdx.size() - 1);
+  /// DWARFLinker doesn't support type units + .debug_names right now anyway,
+  /// so just keeping current behavior.
```

Comment:
Maybe the same wording as the FIXME above?

Reply:
done.